src/NUnitEngine/nunit.engine/Communication/Transports/Tcp/TestAgencyTcpTransport.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Net;
+using System.Threading;
+using NUnit.Common;
+using NUnit.Engine.Internal;
+
+namespace NUnit.Engine.Communication.Transports.Tcp
+{
+    /// <summary>
+    /// Summary description for TestAgencyTcpTransport.
+    /// </summary>
```

Comment:
Would you mind removing the empty summary, please? 🙂

Reply:
I added a good comment here and for a number of other communication classes.

Reply:
Even better! 😁